### PR TITLE
Added rainbow-dark theme

### PR DESCRIPTION
**Pull request by philnik777**
This is a PR for Issue #2810.

### CODE REVIEW

**Comment by RubenRBS** [comment]
For anyone wanting to quickly check how it looks: 
imagen

File: static/colours.scss [hunk 1]
```diff
@@ -169,3 +169,52 @@
...
+    background: rgba(53, 8, 54, 0.35);
+}
+
+.rainbow-dark-10 {
+    background: rgba(24, 97, 8, 0.35);
+}
+
+.rainbow-dark-11 {
+    background: rgba(105, 94, 7, 0.35);
+}
```
Comment (RubenRBS) [inline static/colours.scss:220]: Can you add a new line here? :)

**Review by RubenRBS — APPROVED** [review]
Looking good! Thank you very much for taking the time to add the scheme on a PR :)

**Comment by partouf** [comment]
this is now live

**Pull request merged**